src/main/webapp/user_profile_script.js

```diff
@@ -369,6 +378,56 @@ window.onclick = function(event) {
     }
 }
 
+async function showNotifications() {
+    const request = new Request("/notifications", {method: "GET"});
+    const response = await fetch(request);
+    const notifiResponse = await response.json();
+    const notiList = document.getElementById("notification-list");
+    notiList.innerHTML = "";
+    const noticeContainer = document.getElementById("notice-container");
+    noticeContainer.innerHTML = "";
+    if (notifiResponse.length === 0) return;
+    var totalCnt = 0;
+    for (var index = 0; index < notifiResponse.length; index++) {
+        var li = document.createElement("li");
+        var notification = notifiResponse[index];
+        var overview = notification.overview;
+        var count = notification.count;
+        var id = notification.taskId;
+        // Create a copy for taskId to prevent alias
+        const idCopy = id.slice();
+        totalCnt += count;
+        var taskLink = document.createElement("a");
+        taskLink.setAttribute("class", "notification-element");
+        taskLink.appendChild(
+            document.createTextNode("You have " + count.toString() + " new notificatioins for Task: "
+            + overview));
+        taskLink.addEventListener("click", function() {
+            closeNotificationModal();
+            showTaskInfo(idCopy);
+        });
+        li.appendChild(taskLink);
+        notiList.appendChild(li);
+    }
+    var notiMsg = document.createElement("span");
+    notiMsg.appendChild(document.createTextNode("You have " + totalCnt.toString() + " new notifications"));
+    notiMsg.addEventListener("click", function() {
+        showNotificationList();
+    });
+    notiMsg.setAttribute("class", "blink-text");
```

Comment:
nit: personally I find the blinking a bit distracting, but that's just my personal opinion :)

Reply:
I don't like the blinking text either 🤣 . Until I figure out a better way to inform the users that there are new notifications, I would use plain text to show new notifications.